browser/src/react-redux/App/App.react.jsx

```diff
@@ -1,10 +1,30 @@
-import React from "react";
+import React from 'react';
+import { Route, Switch, Link } from 'react-router-dom';
+
+import SignupPage from './SignupPage';
 
 export default props => {
   return (
     <div>
-      <h1>Dis be my header</h1>
-      <p>Hello</p>
+      <nav>
+        <ul>
+          <li>
+            <Link to='/'>Home</Link>
+          </li>
+          <li>
+            <Link to='/signup'>Sign Up</Link>
+          </li>
+        </ul>
+      </nav>
+
+      <div>
```

Comment:
Thanks for adding the nav! A couple things with the `<Route>`. Let's add this in `App`. Think about a situation in which we won't want to show the nav (possibly a landing page). If our routes are handled in the nav, we lose access to our front end routing unless we _always_ import this nav.